src/components/theme/Navigation/ContextNavigation.stories.js

```diff
@@ -0,0 +1,216 @@
+import ContextNavigationDefault, {
```

Comment:
We are all newbies writing Stories, I guess we will get through and learn as we write them. Having them in MDX is not that important, if we never export them or write prose around enough to be a doc on its own.

Reply:
I need good editor integration with MDX, otherwise it's pretty painful. Maybe I could live with it, let's say, but doing something basic like integrating that style string proved too difficult.